mkldnn.mk

```diff
@@ -39,18 +35,19 @@ mkldnn_build: $(MKLDNN_LIBFILE)
 
 $(MKLDNN_LIBFILE):
 	mkdir -p $(MKLDNNROOT)
-	cd $(MKLDNN_SUBMODDIR) && rm -rf external && cd scripts && ./prepare_mkl.sh && cd .. && cp -a external/*/* $(MKLDNNROOT)/.
-	cmake $(MKLDNN_SUBMODDIR) -DCMAKE_INSTALL_PREFIX=$(MKLDNNROOT) -B$(MKLDNN_BUILDDIR) -DARCH_OPT_FLAGS="-mtune=generic" -DWITH_TEST=OFF -DWITH_EXAMPLE=OFF
+	mkdir -p $(MKLDNNROOT)/lib
+	cmake $(MKLDNN_SUBMODDIR) -DCMAKE_INSTALL_PREFIX=$(MKLDNNROOT) -B$(MKLDNN_BUILDDIR) -DMKLDNN_ARCH_OPT_FLAGS="" -DMKLDNN_BUILD_TESTS=OFF -DMKLDNN_BUILD_EXAMPLES=OFF -DMKLDNN_ENABLE_JIT_PROFILING=OFF
 	$(MAKE) -C $(MKLDNN_BUILDDIR) VERBOSE=1
 	$(MAKE) -C $(MKLDNN_BUILDDIR) install
+	mkdir -p $(MXNET_LIBDIR)
 	if [ -f "$(MKLDNN_LIB64FILE)" ]; then \
-		mv $(MKLDNNROOT)/lib64/libmkldnn* $(MKLDNNROOT)/lib/; \
+		cp $(MKLDNNROOT)/lib64/libmkldnn* $(MXNET_LIBDIR); \
+		cp $(MKLDNNROOT)/lib64/libmkldnn* $(MKLDNNROOT)/lib/; \
+	else \
+		cp $(MKLDNNROOT)/lib/libmkldnn* $(MXNET_LIBDIR); \
 	fi
-	mkdir -p $(MXNET_LIBDIR)
-	cp $(OMP_LIBFILE) $(MXNET_LIBDIR)
-	cp $(MKLML_LIBFILE) $(MXNET_LIBDIR)
-	cp $(MKLDNN_LIBFILE) $(MXNET_LIBDIR)
 	cp $(MKLDNN_BUILDDIR)/include/mkldnn_version.h $(MXNET_INCLDIR)/mkldnn/.
+
 mkldnn_clean:
 	$(RM) -r 3rdparty/mkldnn/build
```

Comment:
Is this still needed?

Reply:
Thanks for reminding us of this. Actually, we can remove the next line `$(RM) -r $(MKLDNNROOT)`.